### PR TITLE
/search/:username + test update

### PR DESCRIPTION
**Pull request by theRJMurray**
**Added:**
-added /search/:username to differentiate between a search and the default root
-reverted npm test to previous form, added /search/:username to pass tests

Working off multiple pull requests not committed to master. Wondering if there will be any problems that pop up because of that.

Realizing that I am going to have to connect this backend to a frontend to display the data how I want it. Deciding between raw html or React.js.

Going to Process JSON Data from project board before I decide.

Forgot to update the readme.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 
-app.get('/:username', function(req, res, error){
+
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname + 'index.html'))
+});
+
+app.get('/search/:username', function(req, res, error){
```
Comment (turtleman270) [inline index.js]: You might want to update the readme to reflect this

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/main.css"/>
+    <title>RuneHelper</title>
+  </head>
+  <body>
+    <h1>Welcome to RuneHelper!</h1>
+    <p>To check your stats, change the url:</p>
+    <p>Example:</p>
+    <p>localhost:3000/INSERT_USERNAME_HERE</p>
```
Comment (turtleman270) [inline public/index.html]: This should also be updated to include search

File: index.js (hunk 1 above)
Comment (turtleman270) [reply index.js]: Does the stuff in `index.html` show up when you hit this route?

**Comment by turtleman270** [comment]
It might have made things easier/ more clear if you only used one branch/PR

You could have committed and pushed additional changes to the original branch instead of creating multiple PRs

File: README.md [hunk 3]
```diff
@@ -4,7 +4,9 @@ Helping people efficiently build their OSRS character
 ## Running the app
 Clone this repository `git clone git@github.com:theRJMurray/runehelper.git`  
 `npm install`  
-`npm start`  
+`npm start`
+
+## Searching User Stats
 In your browser go to localhost:3000/username where username is the
```
Comment (turtleman270) [inline README.md]: 3000/`search`/username

**Comment by theRJMurray** [comment]
Oh I see now. I was thinking that pull requests were immutable for some reason.

**Comment by turtleman270** [comment]
If you merge this, it'll have all the changes in #5 and #6

File: index.js (hunk 1 above)
Comment (theRJMurray) [reply index.js]: yes it does

**Comment by theRJMurray** [comment]
so #5 and #6 can be cancelled, and this one merged?

File: index.js (hunk 1 above)
Comment (turtleman270) [reply index.js]: Then I understand less than I thought I did

**Comment by turtleman270** [comment]
> so #5 and #6 can be cancelled, and this one merged?

I think they might be automatically cancelled because all the commits in those PR's exist in this PR

File: index.js (hunk 1 above)
Comment (theRJMurray) [reply index.js]: No sorry I was mistaken. When you hit the :username route, the index.html does NOT show up. It only shows up when you hit the '/' root.

**Comment by theRJMurray** [comment]
Many lessons learned on that simple pr.

I'll be sure to make a more complete pr next time.

Thanks for the review!

**Pull request merged**